.gitignore

```diff
@@ -143,4 +143,4 @@ venv.bak/
 dmypy.json
 
 # Pyre type checker
-.pyre/
+.pyre/
```

Comment:
Can you add back the newline to this file? There should be a text editor setting for this